src/test/java/com/eppo/sdk/EppoClientTest.java

```diff
@@ -315,23 +315,36 @@ public void testBanditColdStartAction() {
     assertEquals("cold-start-bandit-experiment-bandit", capturedBanditLog.experiment);
     assertEquals("cold-start-bandit", capturedBanditLog.variation);
     assertEquals("subject2", capturedBanditLog.subject);
-    assertEquals(Map.of(), capturedBanditLog.subjectAttributes);
+    assertEquals(Map.of(), capturedBanditLog.subjectNumericAttributes);
+    assertEquals(Map.of(), capturedBanditLog.subjectCategoricalAttributes);
     assertEquals("option3", capturedBanditLog.action);
-    assertEquals(Map.of(), capturedBanditLog.actionAttributes);
+    assertEquals(Map.of(), capturedBanditLog.actionNumericAttributes);
+    assertEquals(Map.of(), capturedBanditLog.actionCategoricalAttributes);
     assertEquals(0.3333, capturedBanditLog.actionProbability, 0.0002);
     assertEquals("cold start", capturedBanditLog.modelVersion);
   }
 
   @Test
   public void testBanditModelActionLogging() {
+    // Note: some of the passed in attributes are not used for scoring, but we do still want to make sure they are logged
```

Comment:
Good test! We definitely want to log "new" attributes, so that they can be used for the next model retrain.